pkg/drivers/powerflex.go

```diff
@@ -375,7 +381,7 @@ func ExtractZonesFromSecret(ctx context.Context, kube client.Client, namespace s
 		}
 		err = yaml.Unmarshal(configs, &yamlConfig)
 		if err != nil {
-			return nil, fmt.Errorf("unable to unmarshal multi-array configuration[%v]", err)
+			return nil, fmt.Errorf("unable to unmarshal no Zone or Name or LabelKey found multi-array configuration[%v]", err)
```

Comment:
Is this right? I don't see labelKey defined in StorageArrayConfig and also zoning configuration is optional (at least for the backward compatibility) - we just need to check that all arrays have consistent configuration.

This function has a few errors formatted as `[%v]`, can you change it to `: %v` for consistency with our conventional formatting.

Reply:
fixed. fixed the [%v] as well.